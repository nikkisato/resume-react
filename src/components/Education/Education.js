import React from 'react';
import data from '../resume2';
import styles from './Education.css';

export default function Education() {
  return (
    <div className={styles.education}>
      <h2>Education</h2>

      <p>
        <b>{data.Education[0].school}</b> -
        {data.Education[0].description[0].endDate}
      </p>
      <p>{data.Education[0].description[0].description}</p>

      <p>
        <b>{data.Education[1].school}</b> -
        {data.Education[1].description[0].endDate}
      </p>
      <p>{data.Education[1].description[0].description}</p>
    </div>
  );
}
