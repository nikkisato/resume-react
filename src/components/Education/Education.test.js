import React from 'react';
import { shallow } from 'enzyme';
import Education from './Education';

describe('Education component', () => {
  it('renders Education', () => {
    const wrapper = shallow(<Education />);
    expect(wrapper).toMatchSnapshot();
  });
});
