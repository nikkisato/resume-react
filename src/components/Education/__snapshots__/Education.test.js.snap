// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Education component renders Education 1`] = `
<div
  className="education"
>
  <h2>
    Education
  </h2>
  <p>
    <b>
      Portland state University
    </b>
     -
    2019
  </p>
  <p>
    BA Advertising and Marketing, Minor in Photography and Design Management
  </p>
  <p>
    <b>
      Alchemy Code Lab
    </b>
     -
    2020
  </p>
  <p>
    Full Stack Developer
  </p>
</div>
`;
